C++/0. Setup.md

```diff
@@ -2,7 +2,12 @@
 
 ## Start
 
-So, in C++ files, there are some things you need to add at the start of the document. `#include <iostream>` includes a package in C++ which lets you use `cout` and `cin` which are the keywords for character output and character input. In some beginner programmer's programs, you may see `using namespace std;` at the starting too, which lets you use a lot of functions without needing to write `std::functionName` all the time. This is useful, but it sometimes includes functions you don't need and you didn't know were there, resulting with you creating a function with the same name which is used in a function in the namespace. The `int main() {}` is where you will put your code. Your code goes inside the curly braces `{}`. This creates a function called main, which is automatically run when you run the file, and the code inside the braces is what will be run.
+In C++ files, one can include other code, such as code for writing and/or reading the console. `#include <iostream>` includes a header in C++ which lets you use `std::cout` and `std::cin` which are the globals variables used for character output and character input. In some beginner programmer's programs, you may see `using namespace std;` at the starting too, which lets you access the standard namespace (`std`) without explicitly writing it in order to access it. This is useful, but it sometimes includes functions you don't need and you didn't know were there, resulting in you creating a function with the same name which is used as a function in the namespace.
+
+
+C++ is a compiled language, as a programmer you write and read textual C++. On it's own, the text does nothing; it has to be compiled. Compilation is the process of generating machine code/binary from a high-level, human-readable language. This process is automated via compilers, programs that compile the text to executable programs.
```

Comment:
Added an explanation as to why you need a main function, unlike, say Python.

Is it too low-level to start off with?

Reply:
what do you mean too low level to start off with? Also Gud jub changing the paragraph. YOU FIRE 🔥

Reply:
I guess it doesn't touch upon low-level concepts at all really.

Just to ask, do you have an expected experience that the reader has? Should it be assumed that thay have read the other tutorials, or have used another language already?

Reply:
Nope. This should all be from the ground up. The only prerequisite should be how to download the compiler and run the code.

Reply:
Got it.